sap/cf_logging/record/simple_log_record.py

```diff
@@ -63,5 +67,9 @@ def format(self):
             'msg': self.getMessage(),
         })
 
+        if record.get('level') == 'ERROR':
```

Comment:
use `self.levelno` to check log level and compare to logging.ERROR constant instead of comparing strings